frontend/packages/operator-lifecycle-manager/src/components/operand.tsx

```diff
@@ -60,7 +60,11 @@ const csvName = () =>
 const getActions = (selectedObj: any) => {
   const actions = plugins.registry
     .getClusterServiceVersionActions()
-    .filter((action) => action.properties.kind === selectedObj.kind);
+    .filter(
+      (action) =>
+        action.properties.kind === selectedObj.kind &&
+        _.startsWith(selectedObj.apiVersion, 'ocs.openshift.io'),
```

Comment:
Not in favor of hardcoding things in core components.

Reply:
ack